tests/PaymentMethodsTest.php

```diff
@@ -1,67 +1,60 @@
 <?php
 
+declare(strict_types=1);
+
 namespace ThePay\ApiClient\Tests;
 
-use Mockery;
 use ThePay\ApiClient\Filter\PaymentMethodFilter;
+use ThePay\ApiClient\Http\HttpServiceInterface;
 use ThePay\ApiClient\Tests\Mocks\Service\ApiMockService;
 use ThePay\ApiClient\TheClient;
 use ThePay\ApiClient\ValueObject\PaymentMethodCode;
 use ThePay\ApiClient\ValueObject\PaymentMethodTag;
 
-class PaymentMethodsTest extends BaseTestCase
+final class PaymentMethodsTest extends BaseTestCase
 {
-    /** @var TheClient */
-    private $client;
+    private TheClient $client;
 
-    /**
-     * @return void
-     */
-    protected function setUp()
+    protected function setUp(): void
     {
         parent::setUp();
 
-        $httpService = Mockery::mock('ThePay\ApiClient\Http\HttpServiceInterface');
-        /** @phpstan-ignore-next-line */
+        $httpService = $this->createMock(HttpServiceInterface::class);
         $apiService = new ApiMockService($this->config, $httpService);
-        /** @phpstan-ignore-next-line */
         $this->client = new TheClient($this->config, null, $httpService, $apiService);
     }
 
-    /**
-     * @return void
-     */
-    public function testGettingActivePaymentMethods()
+    public function testGettingActivePaymentMethods(): void
     {
         $client = $this->getApiaryClient();
 
         $methods = $client->getActivePaymentMethods();
         $cardMethod = $methods->get('card');
 
-        static::assertSame(1, $methods->size());
-        static::assertNotNull($cardMethod);
-        static::assertSame('card', $cardMethod->getCode());
-        static::assertSame('Card Payment', $cardMethod->getTitle());
-        static::assertSame(array('online', 'returnable'), $cardMethod->getTags());
-        static::assertSame(array('CZK'), $cardMethod->getAvailableCurrencies());
-        static::assertSame('https://neco.cz', $cardMethod->getImageUrl()->getValue());
+        self::assertSame(1, $methods->size());
+        self::assertNotNull($cardMethod);
+        self::assertSame('card', $cardMethod->getCode());
+        self::assertSame('Card Payment', $cardMethod->getTitle());
+        self::assertSame(array('online', 'returnable'), $cardMethod->getTags());
```

Comment:
Na pole bych používal jen zkrácený zápis: `['online', 'returnable']`, `['CZK']`. Myslím, že to tak i IDE napovídá.

Reply:
Amazing